### DISCUSSION TITLE
“Last” 7/30 days feature should not include “Today”

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#1021] [Anything Else]
"One thing that definitely feels off and wrong is that the “last” 7/30 days feature should not include “today”! Last implies past and past is not today. If I look at different times of the day and “last 7” changes - that’s weird. That’s specifically important for me, because I don’t care too much about “today” and more about those rolling 7/30 day views"

(feedback via email)

GA doesn't include today either.

**Comment by ukutaht** [comment]
Interesting, it's not something I've thought about. Anybody else have a preference? I don't think I care either way.
